sql-parser/src/test/java/io/crate/sql/parser/TestStatementBuilder.java

```diff
@@ -424,7 +424,14 @@ public void testSetSession() throws Exception {
         printStatement("set session some_setting TO ON");
         printStatement("set session some_setting TO true");
         printStatement("set session some_setting TO 1, ON");
+    }
 
+    @Test
+    public void testInvalidSetSessionValues() throws Exception {
+        expectedException.expect(ParsingException.class);
+        printStatement("set session some_setting = ,value");
+        expectedException.expect(ParsingException.class);
```

Comment:
i think this should not be a parsing exception. you should also remove the according unit test since it actually will not test what really happens.